README.md

```diff
@@ -137,18 +137,48 @@ Events:
 - Status
 - Workflow job
 
-# Development
+# Local development
 
-The bot requires some environment variables listed in
-[config.rs](./src/config.rs). They can, optionally, be set through an `.env`
-file which should be placed at the root of this repository. We provide an
-[example .env file](./.env.example) which can be used as a starting point.
+This project is a standard Rust project with some notable requirements.
+
+## Requiremnets
+
+Before you can generate a debug or release binary, we have some library
+requirements to install.
+
+```sh
+$ sudo apt install \
```

Comment:
It would be better to have this in one line so that someone would be able to copy-and-paste it

Reply:
This is copy-paste-able? I prefer this way for readability